src/components/Avatar/index.js

```diff
@@ -0,0 +1,11 @@
+import {AbstractComponent} from "../AbstractComponent";
+import template from "./template.hbs";
+import noAvatarImage from "../../shared/img/user.svg";
+
+export class Avatar extends AbstractComponent {
+    render() {
+        return template(Object.assign(this.props, {
+            noAvatarImage
+        }));
```

Comment:
Можно использовать деструктуризацию. Код будет выглядеть лаконичней, меньше скобок, лучше читаться
```suggestion
        return template({
            ...this.props,
            noAvatarImage
        });
```